src/metabase/driver/firebolt.clj

```diff
@@ -345,10 +325,6 @@
            ;; find PKs and mark them
            (add-table-pks (.getMetaData conn))))))
 
-; making between clause in a query to be inside brackets
-(defmethod hformat/fn-handler "between" [_ field lower upper]
-  (str "(" (hformat/to-sql-value field) " BETWEEN " (hformat/to-sql-value lower) " AND " (hformat/to-sql-value upper) ")"))
-
```

Comment:
Is this supposed to be removed? Would date ranges still work in this case?

Reply:
yes, this is what Metabase's experts told me.